lib/actions-oclif/internal/osinit.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright 2016-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { stripIndent } from 'common-tags';
+import Command from '../../command';
+import { CommandHelp } from '../../utils/oclif-utils';
+
+// These are internal commands we want to be runnable from the outside.
+// One use-case for this is spawning the minimal operation with root privileges.
```

Comment:
Could anyone elaborate on these notes (taken from old version)? I'm not really sure what these are used for.

Reply:
It sounds like the idea was to run "internal commands" with different privileges (either elevating to root, or dropping root), but my impression was always that it is unnecessary to have "internal commands" for that: it is possible to dynamically gain or drop privileges regardless of internal commands. For code reuse, functions / classes / modules are preferable / simpler than running "internal commands". See the following PR and comments where I deleted "internal sudo":
* https://github.com/balena-io/balena-cli/pull/1455/files
* https://github.com/balena-io/balena-cli/pull/1455#discussion_r334308357
* https://github.com/balena-io/balena-cli/pull/1455#discussion_r334308526

So my impression is that getting rid of internal commands would be a good thing.